src/profiler/columns.tsx

```diff
@@ -163,17 +177,24 @@ export const getColumns = (
   axesOptions: AxesOptions,
   tableOptions: TableOptions,
   smallSize: boolean = false,
+  showPercentage: boolean,
   visibleColumns?: Array<string>
 ): ColumnsType<DataItem | ChildDataItem> => {
   const _visibleColumns = visibleColumns?.length ? visibleColumns : VISISBLE_COLUMNS
 
   const columns = _visibleColumns.map((column, index) => {
+    const isPercentage = showPercentage && COUNT_COLUMNS.includes(column)
+
     const _column: any = {
       title: COLUMNS_TITLES[column] || column,
       dataIndex: column,
       key: column,
       align: LEFT_ALIGN_COLUMNS.includes(column) ? ('left' as 'left') : ('right' as 'right'),
-      render: COLUMNS_WITH_DATES.includes(column) ? renderMeanMinMaxValue : renderValue,
+      render: COLUMNS_WITH_DATES.includes(column)
```

Comment:
1) This would be better called something like ITEM_VALUE_COLUMN_NAMES instead of COLUMNS_WITH_DATES (since we call "item" the thing coming from source data).
2) The "column" here may be better "columnName" since it looks like the _column bellow is something very different then name.
3) Are people used to read the chain of conditional operators? Would normal if statement be more readable?
4) IMPORTANT: Why transform the data at the top? Why simply not pass here the "count" to the render function and we can skip the transformation of the data before.

BTW: We should (in future) rename the "column" coming from my code at https://github.com/stoys-io/stoys/blob/main/spark-du/src/main/scala/io/stoys/spark/dp/DpResult.scala#L14 Perhaps "profile" would be better name. I should have named it better from the beginning. But let's wait with that a bit.

Reply:
1. done
2. done
3. looks good to me personally if used no more that twice. This could be agreed on and included/excluded to the lint config. @AndrewRatnikov  ?
4. I believe It is likely not possible and we need an additional transformation. Ant Table can have render function either by rows or columns. Here we iterate by columns, so basically there is no access to the others' column value (count) inside of a loop